docs/data/joy/components/accordion/accordion.md

```diff
@@ -0,0 +1,58 @@
+---
+product: joy-ui
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">Accordions let users show and hide sections of related content on a page.</p>
+
+:::success
```

Comment:
I'm a bit confused by the callout type. I would expect success as all is great, but the fact that it's not built-in makes we weight a bit more toward info: https://www.notion.so/mui-org/Callouts-and-their-usage-in-the-docs-7e709a02c72142cd8b1a0829861435c5?pvs=4#6ffabde5a39b474bb5d5f91e7e792e90.